AWSPinpoint/AWSPinpointNotificationManager.m

```diff
@@ -257,4 +258,27 @@ - (void)addApplicationStateAttributeToEvent:(AWSPinpointEvent *) event
     }
 }
 
+- (AWSPinpointEvent*)buildEventFromEventType:(NSString *) eventType {
+    AWSPinpointEvent *pushNotificationEvent = [self.context.analyticsClient createEventWithEventType:eventType];
+    [self addApplicationStateAttributeToEvent:pushNotificationEvent
+                         withApplicationState:[[UIApplication sharedApplication] applicationState]];
```

Comment:
I don't think we should call `addApplicationStateAttributeToEvent` from a function called `buildEventFromEventType` -- and if we don't do this, then we should just remove this function.

Please remove this function. I think it adds more confusion by having this function.

Reply:
We do need to add this attribute though. Do you mean just copy the content of `addApplicationStateAttributeToEvent ` into  `buildEventFromEventType `?

Reply:
This is an intermediate step to the next change to add more responsibilities to this `buildEvent` function. Eventually, this function will need to construct eventType string based on userInfo and pushActionType rather than taking it as parameter.

Reply:
Removed `buildEventFromEventType `
Will add this in the next PR when actually building the event